CMakeLists.txt

```diff
@@ -31,5 +31,20 @@ if(NOT IS_LOOKUP_NODE)
     endif()
 endif()
 
+if(NOT WIN32)
+    SET( LevelDB_LIBRARIES leveldb )
+endif()
+if(WIN32)
+    FIND_LIBRARY( LevelDB_LIBRARY libleveldb )
+    SET( LevelDB_LIBRARIES ${LevelDB_LIBRARY} shlwapi )
+endif()
+
+# Dependencies
+FIND_PACKAGE(Boost 1.33 REQUIRED COMPONENTS filesystem system unit_test_framework)
+INCLUDE_DIRECTORIES(${Boost_INCLUDE_DIRS})
+
+FIND_PACKAGE(OpenSSL REQUIRED)
```

Comment:
⚠️ I tested this on Windows, but have not tested it on Ubuntu. Since we don't already have a CI in place for pull requests (e.g. travis-ci), someone should manually verify this.

Reply:
:memo: This appears to be working on Travis CI.